recipe/meta.yaml

```diff
@@ -1,44 +1,48 @@
-{% set version = "1.1.5" %}
+{% set version = "1.1.9" %}
 
 package:
   name: sphinxcontrib-serializinghtml
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/sphinxcontrib-serializinghtml/sphinxcontrib-serializinghtml-{{ version }}.tar.gz
-  sha256: aa5f6de5dfdf809ef505c4895e51ef5c9eac17d0f287933eb49ec495280b6952
+  url: https://pypi.io/packages/source/s/sphinxcontrib-serializinghtml/sphinxcontrib_serializinghtml-{{ version }}.tar.gz
+  sha256: 0c64ff898339e1fac29abd2bf5f11078f3ec413cfe9c046d3120d7ca65530b54
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
+    - wheel
+    - flit-core
   run:
-    - python >=3.5
+    - python
+    - sphinx >=5
 
 test:
-  requires:
-    - pip
-    # This is a circular dependency b/c sphinx depends on this package now :-/
-    - sphinx
-  commands:
-    # pip check incorrectly shows dependencies for sphinx 4.0.1
-    #- pip check
   imports:
     - sphinxcontrib
     - sphinxcontrib.serializinghtml
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
We could easily run the tests: `pytest --pyargs sphinxcontrib.serializinghtml -v`

Reply:
This change has been made

Reply:
Reverted this since pytest fails